opba-auth-rest-impl/src/main/java/de/adorsys/opba/tppauthapi/controller/GenericControllerAdvice.java

```diff
@@ -0,0 +1,52 @@
+package de.adorsys.opba.tppauthapi.controller;
+
+import de.adorsys.opba.tppauthapi.exceptions.PsuAuthenticationException;
+import de.adorsys.opba.tppauthapi.exceptions.PsuAuthorizationException;
+import de.adorsys.opba.tppauthapi.exceptions.PsuRegisterException;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+import java.util.Collections;
+import java.util.List;
+
+@ControllerAdvice(basePackageClasses = {
+        PsuAuthController.class
+})
+@Slf4j
+public class GenericControllerAdvice {
+
+    @ExceptionHandler({PsuRegisterException.class})
+    public ResponseEntity<List<String>> handleUserExistsException(PsuRegisterException ex) {
+        log.debug("User exists exception: {}", ex.getMessage(), ex);
```

Comment:
Increase logging level - it is error events

Reply:
done